3574/1/parse.rb

```diff
@@ -0,0 +1,43 @@
+require 'nokogiri'
+
+class Parser
```

Comment:
main responsibility of this class is to provide information about gems, not parsing. please reflect this in class name